### PR TITLE
Creating .fromRaw API & its tests

### PR DESCRIPTION
**Pull request by LucasHFS**
Attemp to solve #1767 . I'm new to the lib and this is my first PR. Any changes or things to add let me know 😃.

### CODE REVIEW

**Comment by kibertoad** [comment]
@elhigu What do you think?

File: lib/query/querybuilder.js [hunk 1]
```diff
@@ -203,6 +203,17 @@ class Builder extends EventEmitter {
     return this;
   }
 
+  fromRaw(sql, bindings) {
+    //Verify if is rawInstance
```
Comment (elhigu) [inline lib/query/querybuilder.js:207]: why this is not implemented just 

```
const raw = sql.isRawInstance ? sql : this.client.raw(sql, bindings);
return this.from(raw)
```

Comment (kibertoad) [reply lib/query/querybuilder.js:207]: @LucasHFS Ping?..

**Comment by OlivierCavadenti** [comment]
Superseded by #4781

**Pull request closed without merging**